evm/v0.5/contracts/dev/Coordinator.sol

```diff
@@ -294,10 +298,16 @@ contract Coordinator is ChainlinkRequestInterface, CoordinatorInterface {
 
   /**
    * @notice Retrieve the Service Agreement ID for the given parameters
-   * @param _agreement contains all of the terms of the service agreement that can be verified on-chain.
+   * @param _agreementData contains all of the terms of the service agreement that can be verified on-chain.
    * @return The Service Agreement ID, a keccak256 hash of the input params
    */
-  function getId(ServiceAgreement memory _agreement) public pure returns (bytes32)
+  function getId(bytes memory _agreementData) public pure returns (bytes32)
```

Comment:
This is a bit wasteful because we have to build the struct from raw data and then deconstruct it immediately afterwards... there may be an easier way to go straight from `_agreementData` to the `saID`.
Maybe just...
```solidity
keccak256(_agreementData)
```
???

Reply:
Right now, this is only a helper method so I'm inclined to leave it as is instead of designing the hashing and signature around it. Possibly worth moving into a test helper contract in the future, but I'd just leave it as is for now.

Reply:
:+1: